Docs/ProjectSpec.md

```diff
@@ -130,6 +132,84 @@ options:
     tvOS: "10.0"
 ```
 
+### Breakpoints
+
+- [x] **type**: **String** - Breakpoint type
+    - `FileBreakpoint`: file breakpoint
+    - `ExceptionBreakpoint`: exception breakpoint
+    - `SwiftErrorBreakpoint`: swift error breakpoint
+    - `OpenGLErrorBreakpoint`: OpenGL breakpoint
```

Comment:
Could we remove the `Breakpoint` suffix from all these, and just added it when encoding/decoding

Reply:
It looks like it's an easy add in XCProjExtensions as that there's already an init that removes the `Xcode.Breakpoint.` prefix

Reply:
You are right, I will remove the `Breakpoint` suffix.